### PR TITLE
Test:optimize dexmaker

### PR DESCRIPTION
**Pull request by eebssk1**
slightly optimize dexmaker.

### CODE REVIEW

File: dexmaker/build.gradle [hunk 1]
```diff
@@ -14,17 +25,27 @@ dependencies {
     compileOnly project(':dalvikdx')
 }
 
+task proguardFiles(type: ProGuardTask, dependsOn: jar) {
+    verbose
+    forceprocessing
+    configuration files('proguard-rules.pro')
+    libraryjars files('C:/Program Files (x86)/Android/android-sdk/platforms/android-28/android.jar',tasks.getByPath(':dalvikdx:jar').outputs)
```
Comment (kotori2) [inline dexmaker/build.gradle:32]: the hard-coded path looks bad

Comment (eebssk1) [reply dexmaker/build.gradle]: Updated.
I see ANDROID_HOME not redefined in ci script after using a external SDK. However this should work if CI system image already contain Android SDK.

**Comment by kotori2** [comment]
`ANDROID_HOME` is deprecated. https://developer.android.com/studio/command-line/variables . Use `ANDROID_SDK_ROOT` instead.
Also I think this pr will conflict with `android_r` branch. Gonna merge `android_r` into master soon.

**Comment by eebssk1** [comment]
Closed due to long repo desync(I'm too lazy on this pr).

**Pull request closed without merging**